website/docs/docs/build/incremental-strategy.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "About incremental strategy"
 sidebar_label: "About incremental strategy"
-description: "Learn about the various ways (strategies) to implement incremental materializations."
+description: "Implement incremental materializations and the supported dbt adapters."
```

Comment:
What do you think about talking about "optimization" and talking about the new/changed data, which is key to incremental strategy.
```suggestion
description: Use an incremental strategy to optimize performance by defining how incremental materializations handle new and changed data."
```

Reply:
@matthewshaver I like that rewrite but wonder if we should include "materializations". in the title as a good key word. I think Joey mentioned the limit is 150 so we should be good?